test/unit/math/prim/fun/grad_2F1_test.cpp

```diff
@@ -111,37 +155,43 @@ TEST(MathPrimScalFun, grad2F1_9) {
   double z = 0.9;
 
   double grad_a1;
+  double grad_a2;
   double grad_b1;
-  stan::math::grad_2F1(grad_a1, grad_b1, a1, a2, b1, z);
+  stan::math::grad_2F1(grad_a1, grad_a2, grad_b1, a1, a2, b1, z);
   EXPECT_NEAR(0.85880025358111, grad_a1, 1e-8);
+  EXPECT_NEAR(0.4677704416159314, grad_a2, 1e-8);
   EXPECT_NEAR(-4.19010422485256, grad_b1, 1e-8);
 }
 
 TEST(MathPrimScalFun, grad2F1_10) {
   double a1 = 2;
-  double a2 = 1.0;
+  double a2 = 1;
   double b1 = 2;
   double z = 0.4;
 
   double grad_a1;
+  double grad_a2;
   double grad_b1;
-  stan::math::grad_2F1(grad_a1, grad_b1, a1, a2, b1, z);
+  stan::math::grad_2F1(grad_a1, grad_a2, grad_b1, a1, a2, b1, z);
   EXPECT_NEAR(0.4617734323582945, grad_a1, 1e-8);
+  EXPECT_NEAR(0.851376039609984, grad_a2, 1e-8);
   EXPECT_NEAR(-0.4617734323582945, grad_b1, 1e-8);
 }
 
 TEST(MathPrimScalFun, grad2F1_11) {
   double a1 = 3.70975;
-  double a2 = 1.0;
+  double a2 = 1;
   double b1 = 2.70975;
   double z = 0.999696;
 
   double grad_a1;
+  double grad_a2;
   double grad_b1;
-  stan::math::grad_2F1(grad_a1, grad_b1, a1, a2, b1, z);
+  stan::math::grad_2F1(grad_a1, grad_a2, grad_b1, a1, a2, b1, z);
 
   EXPECT_NEAR(29369830.002773938200417693317785, grad_a1,
               1e-1);  // reference: discrete diff in mathematica
+  EXPECT_NEAR(36347869.41885337, grad_a2, 1e-1);
   EXPECT_NEAR(-30843032.10697079073015067426929807, grad_b1,
               1e-1);  // reference: discrete diff in mathematica
 }
```

Comment:
Should we be using the autodiff testing framework for this?

Reply:
I don't think we can because we don't have the 2F1 function, this is - a bit confusingly because it's in prim - the gradient of 2F1.